bin

```diff
@@ -1 +0,0 @@
-node_modules/.bin
```

Comment:
What was this used for previously?

Reply:
I added this as a shortcut for us to directly run installed Node commands, for example, `bin/parcel`.  Note that these are for binscripts that are come with the packages we use, not the scripts we have defined in `package.json`.
- No one ever needed to use it since we set up scripts for them
- I don't think anyone but me actually ran it
- It caused issues like #19
- I realized that Yarn supports a just-as-easy way to do this: `yarn parcel`, etc.